src/views/Play/Play.tsx

```diff
@@ -619,38 +702,38 @@ export class Play extends React.Component<PlayProperties, any> {
         }
     }
 
+    commonSpan = (text: string, align: "center"|"left") => {
+        return <span className="cell" style={{textAlign: align}}>
+            {text}
+        </span>;
+    };
+
     challengeList(isLive: boolean) {
+        const challenge_list = isLive ? this.state.live_list : this.state.correspondence_list;
+
         const user = data.get("user");
 
-        const timeControlClassName = (config) => {
+        const timeControlClassName = (config) => {  // This appears to be bolding live games compared to blitz?
             const isBold = isLive && (config.time_per_move > 3600 || config.time_per_move === 0);
             return "cell " + (isBold ? "bold" : "");
         };
 
-        if (!this.anyChallengesToShow(isLive)) {
+        if (!this.anyChallengesToShow(challenge_list)) {
             return (
                 <div className="ineligible">
                     {this.state.show_all_challenges ?
-                        _("(none)") /* translators: No challenges available */ :
-                        _("(none available)") /* translators: No challenges available */}
+                        _("(none)") /* translators: There are no challenges in the system, nothing to list here */ :
+                        _("(none available)") /* translators: There are no challenges that this person is eligible for */}
                 </div>
             );
         }
 
-        const commonSpan = (text: string, align: "center"|"left") => {
-            return <span className="cell" style={{textAlign: align}}>
-                {text}
-            </span>;
-        };
-
-        const challengeList = isLive ? this.state.live_list : this.state.correspondence_list;
-
-        return challengeList.map((C) => (
+        return challenge_list.map((C) => (
             (((C.eligible || C.user_challenge || this.state.show_all_challenges)
              && ((this.state.show_unranked_challenges && !C.ranked) || (this.state.show_ranked_challenges && C.ranked))
              && ((this.state.show_19x19_challenges && C.width === 19 && C.height === 19) || (this.state.show_13x13_challenges && C.width === 13 && C.height === 13) || (this.state.show_9x9_challenges && C.width === 9 && C.height === 9) || (this.state.show_other_boardsize_challenges && (C.width !== C.height || (C.width !== 19 && C.width !== 13 && C.width !== 9)))) ) ?
-                    <div key={C.challenge_id} className="challenge-row">
-                        <span className="cell" style={{textAlign: "center"}}>
+                    <div key={C.challenge_id} className={"challenge-row" + (C.rengo ? " rengo" : "")}>
```

Comment:
Likewise, if challenge list is guaranteed to be one of `live_list` or `challenge_list`, `C.rengo` will always be false